sdk/src/main/java/io/customer/sdk/api/service/CustomerIOService.kt

```diff
@@ -3,9 +3,10 @@ package io.customer.sdk.api.service
 import io.customer.sdk.api.retrofit.CustomerIoCall
 import io.customer.sdk.data.request.DeviceRequest
 import io.customer.sdk.data.request.Event
+import retrofit2.Response
 import retrofit2.http.*
 
-internal interface CustomerService {
+internal interface CustomerIOService {
```

Comment:
Ah! Now I understand. I am not sure why I renamed this interface. It's `internal` so the naming is not super critical. 

I would prefer a name `CustomerEndpoints` since it just defines all endpoints of the API. You OK with that name?

Reply:
I am actually going to skip this discussion for now if that's OK. I just got done doing a refactor in milestone 4/4 in the background queue. In this refactor, I had to split up the `CustomerService` class and the `PushService` class into [a retrofit service for the tracking module](https://github.com/customerio/customerio-android/blob/7e6b1d6724fd199b6a721e05a4726e48d6c19089/sdk/src/main/java/io/customer/sdk/api/TrackingHttpClient.kt) and [a retrofit service for the messagingpush module](https://github.com/customerio/customerio-android/blob/7e6b1d6724fd199b6a721e05a4726e48d6c19089/messagingpush/src/main/java/io/customer/messagingpush/api/MessagingPushHttpClient.kt).

This is because the logic for push (register push token, delete push token) exists in the messaging-push SDK, not the tracking SDK and I wanted to make each module's retrofit service interface `internal` instead of `public` to avoid exposing them to the customer. 

So, with that change done, this conversation will become irrelevant then. Are you OK with talking about naming then? 

I am also flexible. In fact, I like your idea of staying consistent with retrofit naming since these interfaces are used only for retrofit.

Reply:
Sounds good!